CMakeLists.txt

```diff
@@ -60,6 +61,11 @@ else()
     set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall -Wextra -Wpedantic -Werror")
 endif()
 
+if(CPR_USE_PIC)
+    set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
Any reason, why the `CMAKE_POSITION_INDEPENDENT_CODE` is not being used here?

https://cmake.org/cmake/help/latest/variable/CMAKE_POSITION_INDEPENDENT_CODE.html

Reply:
Excuse me for not knowing this variable.

I'll change it later.